tests/test_oauth1_session.py

```diff
@@ -95,6 +109,11 @@ def fake_send(r, **kwargs):
     @mock.patch('oauthlib.oauth1.rfc5849.generate_timestamp')
     @mock.patch('oauthlib.oauth1.rfc5849.generate_nonce')
     def test_signature_methods(self, generate_nonce, generate_timestamp):
+        try:
+            import cryptography
+        except ImportError:
+            raise unittest.SkipTest('cryptography not available')
```

Comment:
Rather than doing this four-line check, I'd suggest doing this at the top of the file:
```python
try:
    import cryptography
except ImportError:
    cryptography = None
```
and the using the [`skipUnless()`](https://docs.python.org/2/library/unittest.html#unittest.skipUnless) decorator on tests that require the `cryptography` module:
```python
@unittest.skipUnless(cryptography, "cryptography module is required")
def test_signature_methods(self, generate_nonce, generate_timestamp):
    # ...
```